src/models/stackedAreaChart.js

```diff
@@ -41,7 +40,7 @@ nv.models.stackedAreaChart = function() {
     xAxis.orient('bottom').tickPadding(7);
     yAxis.orient((rightAlignYAxis) ? 'right' : 'left');
 
-    tooltip
+    interactiveLayer.tooltip
```

Comment:
This seems like a logical next step based on the previous feedback (that users should call `chart.interactiveLayer.tooltip` instead of `chart.tooltip` when formatting tooltips).

This edit is using the previous defaults for `tooltip` as the new defaults for `interactiveLayer.tooltip`, though if necessary, the defaults for `tooltip` can be left in as well.